src/HindsightLogChannel.php

```diff
@@ -4,6 +4,9 @@
 
 use Monolog\Logger;
 
+/**
+ * @deprecated The Hindsight log channel has been deprecated in favour of the Hindsight log driver as of version 0.3
```

Comment:
We're still 0.x, is it worth keeping things deprecated? I think people can expect breaking changes pre-1.0.0. Or will this be removed at 1.0.0?

Reply:
Just wanted to deprecate it to make it as easy as possible for customers, and then remove in the following release